sr_utilities_common/src/sr_utilities_common/awsmanager.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
```

Comment:
why do we have 2 files? I think it's better to just modify the original file and keep one. we can have a call regarding that.

Reply:
Corrected, awsmanager.py removed and aws_manager (with class and previous main) moved to /src